### PR TITLE
delay updates until we've stopped changing for a while.

### PR DESCRIPTION
**Pull request by sdague**
This is a super rough and early attempt to fix bug #230 

Definitely not ready in the current state (not tested), but a discussion point on direction.

### CODE REVIEW

File: src/components/ha-climate-control.html [hunk 1]
```diff
@@ -64,6 +64,7 @@
       },
       incrementValue: function () {
         var newval = this.value + this.step;
+        this.last_changed = new Date();
```
Comment (balloob) [inline src/components/ha-climate-control.html]: Prefer `Date.now()`

**Review by balloob — APPROVED** [review]
I think that this is a pretty ok first pass. (if it works)

For the next version, we can use the [`pressed`](https://www.webcomponents.org/element/PolymerElements/paper-icon-button/paper-icon-button#property-pressed) property on the `paper-icon-button` to keep firing events while user is holding down it's finger. We used to have similar logic in HASS for media player volume but I think we removed it. PR with the source is [here](https://github.com/home-assistant/home-assistant-polymer/pull/17/files#diff-9711b10ae2ec098d828bb798920dccddR171)

**Comment by sdague** [comment]
This is tested now, probably good enough for the next release. I'll look at the pressed logic as a follow on.

**Comment by sdague** [comment]
I do wonder if there is a better pattern than the window.setTimeout one

**Comment by armills** [comment]
Can you try Polymer debounce? https://www.polymer-project.org/1.0/docs/devguide/instance-methods#async-and-debounce

**Comment by balloob** [comment]
Polymer debounce under the hood will use setTimeout ;-)

**Pull request merged**